src/Components/Wishlist/WishItem.jsx

```diff
@@ -22,10 +22,14 @@ export default function WishItem({ wishItem, userDispatch }) {
           ₹ {originalPrice}
         </span>
       </p>
-      <span className="rating-block theme-shade-2">
-        {rating}
-        <i className="fa fa-star fa-md rating-checked"></i>
-      </span>
+      <div>
+        <span
+          className="rating-block txt-white"
+          style={{ backgroundColor: rating >= 3 ? "#26a541" : "#ff6161" }}
```

Comment:
My suggestion, you can create a variable before return and call here for more readability.

Reply:
Okay, I will change it